libs/RobotKit/include/RobotController.h

```diff
@@ -294,6 +305,11 @@ class RobotController : public interface::RobotController
 		stopActuators();
 	}
 
+	void turnOffHardware() final
```

Comment:
```suggestion
	void suspendHardwareForDeepSleep() final
	{
		log_info("Entering deep sleep");
	}
```

Reply:
Note. logs is not necessary since SM already display it

Reply:
yes I know, but SM displays the transitions not the "actual work".

the idea is just to have something print that we'll replace with the real implementation later to make sure the function is called when running.

```suggestion
	void suspendHardwareForDeepSleep() final
	{
		log_info("TO IMPLEMENT - configuring hardware for deepsleep");
	}
```